CONTRIBUTING.md

```diff
@@ -13,13 +13,37 @@ pnpm run setup
 
 Note for Windows: Use the latest version of [Git Bash](https://gitforwindows.org/)
 
+### General prerequisites
+
+1. Install [`pnpm`](https://pnpm.js.org/) (for installing npm dependencies)
+1. Install [`docker`](https://www.docker.com/products/docker-desktop) (for managing test databases)
+
 ### Developing Prisma Client JS
 
-2. `cd src/packages/client`
-3. `ts-node fixtures/generate.ts ./fixtures/blog/ --skip-transpile`
-4. `cd fixtures/blog`
-5. `prisma migrate save --name init --experimental && prisma migrate up --experimental`
-6. `ts-node main.ts`
+1. `cd src/packages/client`
+1. `ts-node fixtures/generate.ts ./fixtures/blog/ --skip-transpile`
+1. `cd fixtures/blog`
+1. `export DB_URL=YOUR_POSTGRES_DB_URL`  
+   For this step you might find our [docker-compose setup](./src/docker) helpful
+1. `npx prisma migrate save --name init --experimental && prisma migrate up --experimental`
```

Comment:
```suggestion
1. `npx @prisma/cli migrate save --create-db --name init --experimental && npx @prisma/cli migrate up --experimental`
```

So it works even if you don't have a Prisma CLI isntalled globally

Reply:
Ah yep!